### PR TITLE
CE-2690 Create an entry point to archives of reviews from a wikia name

### PR DESCRIPTION
**Pull request by adamkarminski**
**DO NOT MERGE BEFORE https://github.com/Wikia/app/pull/8479**

This PR makes an entry point to an archive of completed reviews for a given wikia from the wikia's name on the _Special:ContentReview_ page.

@kamilkoterba

### CODE REVIEW

File: extensions/wikia/ContentReview/controllers/templates/ContentReviewSpecialController_archive.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<h1 class="content-review-special-header-content-title">
+	<?= wfMessage( 'content-review-special-title' )->escaped() ?>
+</h1>
+<div class="content-review-special-back-link">
```
Comment (kamilkoterba) [inline extensions/wikia/ContentReview/controllers/templates/ContentReviewSpecialController_archive.php:4]: You may want to fill this placeholder

**Comment by kamilkoterba** [comment]
LGTM

File: extensions/wikia/ContentReview/controllers/templates/ContentReviewSpecialController_archive.php (hunk 1 above)
Comment (adamkarminski) [inline extensions/wikia/ContentReview/controllers/templates/ContentReviewSpecialController_archive.php:4]: Done!

**Comment by kamilkoterba** [comment]
:+1:

**Pull request merged**